cfg/osic1p.cfg

```diff
@@ -7,7 +7,7 @@ SYMBOLS {
 }
 MEMORY {
     # for size of ZP see runtime/zeropage.s and c1p/extzp.s
-    ZP:       file = "", define = yes, start = $0002, size = $001A + $0004;
+    ZP:       file = "", define = yes, start = $0002, size = $001A + $0005;
```

Comment:
I had thought that it would be a good thing not reserve unused memory here. I will add some extra space the next time I add a variable.